src/cli.ts

```diff
@@ -32,8 +32,19 @@ export async function run() {
         type: "string",
         desc: "A git tag that determines the upper bound of the range of commits",
       },
+      "next-version": {
+        type: "string",
+        desc: "The name of the next version",
```

Comment:
this should probably specify a default of `Unreleased`

Reply:
done